### PR TITLE
c2ea: slightly less stupid edition

### PR DESCRIPTION
**Pull request by StanHash**
PFinder is hella slow, and unless for some reason you're changing base ROMs each time you assemble your buildfile (don't do that), calling it every time you build your events is pointless.

So instead of putting in `#inctext PFinder ...` I just put something akin to the PFinder output directly into the generated table event file.

This can make the conventional `MAKE HACK_quick.cmd` seconds faster.

### CODE REVIEW

File: c2ea.py [hunk 1]
```diff
@@ -2,12 +2,11 @@
 import tkinter as tk
 from tkinter import filedialog
 
-
 def show_exception_and_exit(exc_type, exc_value, tb):
-    import traceback
-    traceback.print_exception(exc_type, exc_value, tb)
-    input("Press Enter key to exit.")
-    sys.exit(-1)
+  import traceback
```
Comment (Crazycolorz5) [inline c2ea.py]: I think this block should stay at 4 indent (And throughout-- I think we should keep 4 space indents).

**Comment by Crazycolorz5** [comment]
@boviex Don't merge this yet; I'm going to make a PR to this branch to improve it even more.

File: c2ea.py (hunk 1 above)
Comment (Crazycolorz5) [reply c2ea.py]: I'll handle this

**Comment by StanHash** [comment]
@boviex Colorz went crazy and now this is ready.

**Comment by Crazycolorz5** [comment]
Sorry for not going through a formal pull request, but the last two commits just make when cache is loaded/written smarter to avoid unnecessary work/conflicts when the program is called in parallel (say, by make).

Edit: Confirmed it does not change the assembly of VBA2.

**Pull request merged**